spec.html

```diff
@@ -38868,7 +38874,7 @@ <h1>%AsyncFromSyncIteratorPrototype%.return ( _value_ )</h1>
         </emu-clause>
 
         <emu-clause id="sec-%asyncfromsynciteratorprototype%.throw">
-          <h1>%AsyncFromSyncIteratorPrototype%.throw ( _value_ )</h1>
+          <h1>%AsyncFromSyncIteratorPrototype%.throw ( [ _value_ ] )</h1>
```

Comment:
Can we add a NOTE explaining that this optional argument is, in fact, always passed? (Since `AsyncFromSyncIteratorPrototype ` objects are not user-observable and the only in-spec [callsite](https://tc39.es/ecma262/#sec-generator-function-definitions-runtime-semantics-evaluation), in `yield *`, always passes an argument.)

Reply:
Sure, but can you help me understand the value of noting that when the entire `% AsyncFromSyncIteratorPrototype%` object is already unobservable?

Reply:
_Because_ it is unobservable, it's basically an abstract operation. By comparison, all abstract operations are unobservable, but it would still be surprising to define an abstract operation which took an optional parameter which was, in fact, always passed.

Here I don't really mind marking the parameter as optional and having logic for what to do if the argument is not passed (though now that I write this I guess I'd be happier still if we just asserted it were, but whatever), but since it's surprising, I think it would benefit from a NOTE.

Reply:
Note added.